src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java

```diff
@@ -141,21 +135,46 @@ private void verifyDeletedFieldPresentInValue(long recordsCount, YBExtractNewRec
         }
     }
 
+    /**
+     * Consume the records available and add them to a list for further assertion purposes.
+     * @param records list to which we need to add the records we consume, pass a
+     * {@code new ArrayList<>()} if you do not need assertions on the consumed values
+     * @param recordsCount total number of records which should be consumed
+     * @param milliSecondsToWait duration in milliseconds to wait for while consuming
+     */
+    private void waitAndFailIfCannotConsume(List<SourceRecord> records, long recordsCount,
+                                            long milliSecondsToWait) {
+        AtomicLong totalConsumedRecords = new AtomicLong();
+        long seconds = milliSecondsToWait / 1000;
+        try {
+            Awaitility.await()
+                .atMost(Duration.ofSeconds(seconds))
```

Comment:
Is a polling interval also required?

Reply:
We can add it, but it may unnecessarily delay the test execution time.